gridwords.py

```diff
@@ -162,25 +169,25 @@ def getColor(self):
 # Create a CellGrid class that derives from tk.Frame
 # it will have a grid of Cells
 class CellGrid(tk.Frame):
-    # define the ctor method 
+    # define the ctor method
     def __init__(self, master=None):
         # initialize the base class
         tk.Frame.__init__(self, master, bg=BLACK, bd=CELL_MARGIN)
         global frames_dict
         frames_dict["cell_grid"] = self
-        
+
         # set mode
         self.mode = 'grid'
-        
```

Comment:
I'm removing some pure whitespace.  Such lines show up in my editor and poke me in the eye.